org.eclipse.jdt.apt.pluggable.tests/src/org/eclipse/jdt/apt/pluggable/tests/processors/buildertester/Issue565Processor.java

```diff
@@ -35,14 +34,12 @@ public class Issue565Processor extends AbstractProcessor {
 
 	@Override
 	public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+        if (roundEnv.processingOver()) {
+            // We're not interested in the postprocessing round.
+            return false;
+        }
 
-		String classpath = System.getProperty("java.class.path");
-		String[] classPathValues = classpath.split(File.pathSeparator);
-		for (String classPath: classPathValues) {
-			System.out.println(classPath);
-		}
-
-		for (Element element : roundEnv.getElementsAnnotatedWith(annotations.stream().findAny().get())) {
+		for (Element element : roundEnv.getElementsAnnotatedWith(Annotation565.class)) {
```

Comment:
This is causing a compilation problem. The previous change was better, imo. Also, please correct the indentation, squash all the changes if possible. And please mark all the resolved comments as such.